### PR TITLE
US 1583733 Add missing language IDs - 22

### PR DESCRIPTION
**Pull request by v-thpra**
- Used "console" for all Framework tool syntax and command examples.
- Used "csharp" for examples of an assembly and a type library.

### CODE REVIEW

File: docs/framework/tools/tlbexp-exe-type-library-exporter.md [hunk 1]
```diff
@@ -68,14 +68,14 @@ tlbexp assemblyName [options]
   
  If you use the <xref:System.Runtime.InteropServices.MarshalAsAttribute> attribute to specify a <xref:System.Runtime.InteropServices.MarshalAsAttribute.SafeArraySubType> value of `VT_UNKOWN` or `VT_DISPATCH`, Tlbexp.exe ignores any subsequent use of the <xref:System.Runtime.InteropServices.MarshalAsAttribute.SafeArrayUserDefinedSubType> field. For example, given the following signatures:  
   
-```  
+```csharp
```
Comment (rpetrusha) [inline docs/framework/tools/tlbexp-exe-type-library-exporter.md]:
```suggestion
```cpp
```

File: docs/framework/tools/tlbexp-exe-type-library-exporter.md [hunk 2]
```diff
@@ -68,14 +68,14 @@ tlbexp assemblyName [options]
...
-```  
+```csharp  
 [return:MarshalAs(UnmanagedType.SafeArray, SafeArraySubType=VarEnum.VT_UNKNOWN, SafeArrayUserDefinedSubType=typeof(ConsoleKeyInfo))] public Array StructUnkSafe(){return null;}  
 [return:MarshalAs(UnmanagedType.SafeArray, SafeArraySubType=VarEnum.VT_DISPATCH, SafeArrayUserDefinedSubType=typeof(ConsoleKeyInfo))] public Array StructDispSafe(){return null;}  
 ```  
   
  the following type library is generated:  
   
-```  
+```csharp
```
Comment (rpetrusha) [inline docs/framework/tools/tlbexp-exe-type-library-exporter.md]:
```suggestion
```cpp
```

File: docs/framework/tools/tlbimp-exe-type-library-importer.md [hunk 3]
```diff
@@ -81,31 +81,31 @@ tlbimp tlbFile [options]
...
   
-```  
+```console  
 tlbimp TestLib.dll /transform:dispret /out:myTestLib.dll  
 ```  
   
  The type library `TestLib.dll`, in the preceding example, includes a dispinterface method named `SomeMethod` that returns void and has an [out, retval] parameter. The following code is the input type library method signature for `SomeMethod` in `TestLib.dll`.  
   
-```  
+```console
```
Comment (rpetrusha) [inline docs/framework/tools/tlbimp-exe-type-library-importer.md]:
```suggestion
```cpp
```

File: docs/framework/unmanaged-api/guid-managedname-attribute.md [hunk 4]
```diff
@@ -35,7 +35,7 @@ Defines a custom interface attribute that specifies the managed namespace name f
 ## Definition  
  `GUID_ManagedName` is defined in Cor.h as follows:  
   
-```  
+```csharp
```
Comment (rpetrusha) [inline docs/framework/unmanaged-api/guid-managedname-attribute.md]:
```suggestion
```cpp
```

File: docs/framework/unmanaged-api/guid-managedname-attribute.md [hunk 5]
```diff
@@ -50,7 +50,7 @@ EXTERN_GUID(GUID_ManagedName, 0xf21f359, 0xab84, 0x41e8, 0x9a, 0x78, 0x36, 0xd1,
 ## Example  
  The following example shows a library definition using the `GUID_ManagedName` attribute.  
   
-```  
+```csharp
```
Comment (rpetrusha) [inline docs/framework/unmanaged-api/guid-managedname-attribute.md]:
```suggestion
```idl
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for adding these language identifiers, @v-thpra. I've suggested some changes.

File: docs/framework/unmanaged-api/guid-managedname-attribute.md (hunk 5 above)
Comment (v-thpra) [reply docs/framework/unmanaged-api/guid-managedname-attribute.md]: Thanks for reviewing these, Ron!

File: docs/framework/tools/tlbexp-exe-type-library-exporter.md [hunk 6]
```diff
@@ -68,14 +68,14 @@ tlbexp assemblyName [options]
   
  If you use the <xref:System.Runtime.InteropServices.MarshalAsAttribute> attribute to specify a <xref:System.Runtime.InteropServices.MarshalAsAttribute.SafeArraySubType> value of `VT_UNKOWN` or `VT_DISPATCH`, Tlbexp.exe ignores any subsequent use of the <xref:System.Runtime.InteropServices.MarshalAsAttribute.SafeArrayUserDefinedSubType> field. For example, given the following signatures:  
   
-```  
+```cpp
```
Comment (rpetrusha) [inline docs/framework/tools/tlbexp-exe-type-library-exporter.md]:
```suggestion
```csharp  
```

File: docs/framework/unmanaged-api/guid-managedname-attribute.md [hunk 7]
```diff
@@ -22,7 +22,7 @@ Defines a custom interface attribute that specifies the managed namespace name f
   
 ## Syntax  
   
-```  
+```csharp
```
Comment (rpetrusha) [inline docs/framework/unmanaged-api/guid-managedname-attribute.md]:
```suggestion
```idl
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Sorry, Tom. I seem to have missed two

**Comment by v-thpra** [comment]
Made the changes. Thanks for catching those, Ron.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks for the changes, Tom. I'll merge your PR now.

**Pull request merged**